lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -176,6 +190,11 @@ sub create_from_settings {
     # this will associate currently available assets with job
     $job->register_assets_from_settings;
 
+    if (%group_args && !$group) {
+        OpenQA::Utils::log_warning(
+            'Ignoring invalid group ' . to_json(\%group_args) . ' when creating new job ' . $job->id);
```

Comment:
JSON is also used at least in the fedmsg plugin. Data::Dumper might insert newlines but I want to prevent that (the hash should fit into the line as it will be just `{id/name: value}`).